deps/url.lua

```diff
@@ -25,22 +25,28 @@ function exports.parse(url, parseQueryString)
   local chunk, protocol = url:match("^(([a-z0-9+]+)://)")
   url = url:sub((chunk and #chunk or 0) + 1)
 
-  local chunk, auth = url:match('(([0-9a-zA-Z]+:?[0-9a-zA-Z]+)@)') 
+  chunk, auth = url:match('(([0-9a-zA-Z]+:?[0-9a-zA-Z]+)@)') 
```

Comment:
Thanks! Good to know about your workflow and tools. I was used vim for my lua coding. I will give a try to sublimelinter and luacheck.